generators/entity-client/templates/angular/src/test/javascript/spec/app/entities/_entity-management-delete-dialog.component.spec.ts

```diff
@@ -0,0 +1,93 @@
+<%#
+ Copyright 2013-2017 the original author or authors from the JHipster project.
+
+ This file is part of the JHipster project, see http://www.jhipster.tech/
+ for more information.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-%>
+<%_
+const tsKeyId = generateTestEntityId(pkType, prodDatabaseType);
+_%>
+/* tslint:disable max-line-length */
+import { ComponentFixture, TestBed, async, inject, fakeAsync, tick } from '@angular/core/testing';
+import { NgbActiveModal } from '@ng-bootstrap/ng-bootstrap';
+import { Observable } from 'rxjs/Rx';
+import { JhiDateUtils, JhiDataUtils, JhiEventManager } from 'ng-jhipster';
+import { <%=angularXAppName%>TestModule } from '../../../test.module';
+import { MockEventManager } from '../../../helpers/mock-event-manager.service';
+import { MockActiveModal } from '../../../helpers/mock-active-modal.service';
+import { <%= entityAngularName %>DeleteDialogComponent } from '../../../../../../main/webapp/app/entities/<%= entityFolderName %>/<%= entityFileName %>-delete-dialog.component';
+import { <%= entityAngularName %>Service } from '../../../../../../main/webapp/app/entities/<%= entityFolderName %>/<%= entityFileName %>.service';
+
+describe('Component Tests', () => {
+
+    describe('<%= entityAngularName %> Management Delete Component', () => {
+        let comp: <%= entityAngularName %>DeleteDialogComponent;
+        let fixture: ComponentFixture<<%= entityAngularName %>DeleteDialogComponent>;
+        let service: <%= entityAngularName %>Service;
+        let mockEventManager: any;
+        let mockActiveModal: any;
+
+        beforeEach(async(() => {
+            TestBed.configureTestingModule({
```

Comment:
same here, can we have a single common file for all entities and then override it with a custom file for each entity group.